src/engine/strat_engine/blockdev.rs

```diff
@@ -55,10 +56,67 @@ pub fn resolve_devices(paths: &[&Path]) -> io::Result<BTreeSet<Device>> {
     Ok(devices)
 }
 
+#[derive(Debug)]
+pub struct BlockDevs {
```

Comment:
I think that BlockDevs could probably be better implemented as a module then a struct + impl.